src/main/java/com/gluonhq/richtext/viewmodel/RichTextAreaViewModel.java

```diff
@@ -313,4 +333,9 @@ private void lineEnd() {
         int pos = getNextRowPosition.apply(Double.MAX_VALUE, false);
         setCaretPosition(pos);
     }
+
+    private void updateStackSize() {
```

Comment:
The method name suggests that we are updating the stack size where as we are just updating the property referring to its state.